src/common/storage/collection_and_field_names.py

```diff
@@ -17,8 +17,8 @@
 FLD_ARANGO_KEY = "_key"
 FLD_ARANGO_ID = "_id"
 
-FLD_COLLECTION_NAME = "coll"
-""" The name of the key that has a collection name as its value. """
+FLD_COLLECTION_ID = "coll"
+""" The name of the key that has a collection ID as its value. """
```

Comment:
I realized I've been using both name and ID to refer to the collection ID. Since collections have both names and IDs, this is pretty confusing.